### ISSUE TITLE
tfenv uninstall not found (homebrew installation)

### ISSUE THREAD

**Issue by ecbrodie** [kamatama41/tfenv#54]
I have tfenv version 0.4.3 installed via homebrew on my macOS 10.12.5 machine. I am trying to use the `tfenv uninstall` command, as documented on your README. But, it looks like the command is not found:

```
$ tfenv uninstall 0.9.8
no such command 'uninstall'
Usage: tfenv <command> [<options>]

Commands:
   install       Install a specific version of Terraform
   use           Switch a version to use
   list          List all installed versions
   list-remote   List all installable versions
```

Please assist. Thank you.

**Comment by ecbrodie** [comment]
This will likely be solved by #55

**Comment by kamatama41** [comment]
Hi, thank you for the report. I'll release new version.
If you need the uninstall command urgently, please try installing tfenv manually (= git clone and add tfenv to your `PATH`) or add `--HEAD` option to `brew install tfenv` (= `brew install tfenv --HEAD`).

**Comment by ecbrodie** [comment]
Thanks @kamatama41 . Since this will be closed by #55 (which requests the new release), maybe it makes sense to close this issue and report the status of the release in the other issue? Up to you.
